app/controllers/api/v1/opportunities_controller.rb

```diff
@@ -9,51 +7,28 @@ def index
 
   # GET /opportunities/:id
   def show
-    @opportunity = Opportunity.where(salesforce_id: params[:id])
+    @opportunity = Opportunity.find_by!(salesforce_id: params[:id])
     render json: @opportunity, status: :ok
   end
 
   # POST /opportunities(.:format)
   def create
-    push_opportunity = PushOpportunityToSalesforce.new
-    sf_opportunity = push_opportunity.create_new_opportunity(opportunity_params)
-    if !sf_opportunity.blank? && !sf_opportunity.id.blank?
-      @opportunity = Opportunity.new(opportunity_params)
-      # setting the value this way because of issue with the Salesforce field named 'type'
-      @opportunity.update_type = 'New Business'
-      @opportunity.salesforce_id = sf_opportunity.id
-      if @opportunity.save
-        render json: { opportunity_id: @opportunity.salesforce_id, status: 'SFAPI and SF Opportunity creation status: Success' }
-      else
-        render json: { opportunity_id: sf_opportunity.id, status: 'SFAPI Opportunity creation status: Failure. SF Opportunity creation status: Success' }
-      end
-    end
+    puts opportunity_params
```

Comment:
I think you want to remove this line

Reply:
done, thanks! I was trying to figure out where this was coming from when I ran the tests 😅